docs/serverless-demo.md

```diff
@@ -1,28 +1,26 @@
-# Welcome to the Apollo Showcase Demo
+# Welcome to the **{{ graph.name }}** API! 🎉
 
-Welcome to the **{{ graph.name }}** API! 🎉 You can follow along in this README to walk through a guided demo or just explore yourself.
+You can follow along in this README to walk through a guided demo or just explore around yourself.
 
-> If you're new to Apollo, learn more about GraphOS in [our docs](https://www.apollographql.com/docs/studio/).
+# So what did clicking "Try a Demo Graph" do?
 
-## So what did clicking that button do to get me here?
+Behind the scenes, we've set up GraphOS to work with two GraphQL APIs we have hosted for you:
 
-In creating this demo, we've set up GraphOS to work with two GraphQL APIs we have hosted for you:
-
-- _spotify_: Exposes the Spotify REST API
+- _spotify_: Exposes the [Spotify REST API](https://developer.spotify.com/documentation/web-api)
 - _playback_: Exposes subscriptions for playback functionality coming from the Spotify REST API including the associated `mutation` operations
```

Comment:
This description is a mouthful but I didn't change it because I wasn't sure which were the important bits. Can we simplify to just "Spotify playback functionality"?